packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -87,6 +87,10 @@ type Props = {|
    * Disable dates outside a min date. See the [delimited selection period example](https://gestalt.pinterest.systems/datepicker#maxMinDates) to learn more.
    */
   minDate?: Date,
+  /**
+   * A unique name for the input.
+   */
+  name: string,
```

Comment:
Should this be required? If so, can you add a note about the relevant codemod in the PR summary?

Reply:
Actually, this was a bug. it's just optional all the way